Base/QTCore/qSlicerCoreIOManager.cxx

```diff
@@ -776,6 +782,7 @@ bool qSlicerCoreIOManager::saveNodes(qSlicerIO::IOFileType fileType,
       continue;
       }
     nodes << writer->writtenNodes();
+    emit fileSaved(parameters);
```

Comment:
similarly to what is done here:

https://github.com/Slicer/Slicer/blob/e5c5ad7801b81c99a0e37b44ce570e9acbfdb7f0/Base/QTCore/qSlicerCoreIOManager.cxx#L611

I suggest to introduce `savedFileParameters` and explicitly set the file type.

Reply:
There are several other parameters that are used for the file loading (coordinate system, centered, etc). They all need to be saved. Scene saving is a special case, we know that we don't lose information by sanitizing the properties, but in general we need to pass all parameters. So, I'm not exactly sure what you mean to change.